### ISSUE TITLE
Failing to scale on 800x480 LCD screen

### ISSUE THREAD

**Issue by AmbassadorHazy** [JuliusCode/MP4MUSEUM#15]
Hi - I'm an artist totally new to Pi and bought a Zero specifically to use MP4Museum. I've gotten it working beautifully on a monitor via HDMI cable - it's super elegant, and thanks for sharing it!

I want to connect a 5" LCD screen, 800x480, DPI interface via LCD hat on the Pi Zero. The [company's documentation](https://www.waveshare.com/wiki/5inch_LCD_for_Pi) recommends changes to the config.txt file, but those suggested for a OS Buster branch match those you've already supplied in the file, so I don't see a place to update. 

Without changes, the player boots up and loops my video, but it does not scale at all - I see only the upper left corner of your intro video and my looped media. I've reviewed your documentation and tried to uncomment areas like those that force a resolution of 800x480. Am I missing something obvious? Because the LCD powers up at the same time as the Pi, is the Pi not recognizing the screen size and instead assuming it's larger?

Not sure this is an issue vs my own ignorance, but thought I'd submit anyway. Thanks!

**Comment by JuliusCode** [comment]
Hi there,

i would suggest to empty out config.txt and just start with the lines they provide for the buster branch. if the display works, you could put the first lines from the mp4museum config.txt back in, if you need them at all.

best, Julius.

**Comment by AmbassadorHazy** [comment]
Hi Julius:

Thanks for your response. I’ve run tests with different configurations - including lines provided by Waveshare, with and without your opening lines from the mp4museum config.txt file as you suggest. I also tried different ways to force the 800x480, but never get the screen content to rescale down from the upper left corner. Photo of what I am seeing pasted here. 

I re-wrote the SD card with the Raspberry Pi OS (Legacy) and by adding the manufacturer code to the config.txt file I can boot up and see the Pi desktop at correct resolution. I then rewrote mp4museum to the card and replaced config.txt with the one that displayed the desktop, but it reverted to the unscaled video. 

My assumption is that there is something in the config.txt file that is in conflict with the files you’ve included in the .img file, but I am stumped as to what to do next.

Thanks,
Thomas
![IMG_0828](https://user-images.githubusercontent.com/101848576/159054520-361179d6-c661-4881-ba9a-7b9dd72a61c2.JPG)

**Comment by JuliusCode** [comment]
so one more thing you could try is to manually set the screenbuffer size with framebuffer_width and framebuffer_height

if that is not doing it you could try the other method for buster, but i don‘t know if these things they are talking about is actually included in the lite version, they make no mention of it.

i had cases in the past with displays like yours where it was just sufficient to specify the timing for the display in config.txt, but for the new os and/or your display, it‘s no longer working.

please do share any info on how this works out.

best, Julius

**Comment by AmbassadorHazy** [comment]
Thank you again. I did try the screen buffer size settings but did not get that to work. I did find a (maybe obvious) solution, however. My original video was rendered in 1920x1080 and would not scale. I simply rerendered it in 800x480 and it played fine. Your intro/splash is still not scaled, but the media I've supplied is, and of course that's the aim. Apologies for taking up your time if this is an obvious thing I should have thought of. Appreciate your supplying MP4Museum - exactly what I was looking for!

-Thomas
